main.py

```diff
@@ -58,22 +62,23 @@ def main():
             loss.backward()
             opt.step()
             epoch_loss += loss.item()
-            break  # TODO: Delete this
+            # break  # TODO: Delete this
 
         # Compute epoch losses
         epoch_loss = epoch_loss / num_training_sequences
         training_losses.append(epoch_loss)
         epoch_val_loss, val_iter = compute_validation_loss(args, model, datasets, criterion, device, mask)
         validation_losses.append(epoch_val_loss / ((val_iter + 1) * batch_size))
+        print(f"Epoch {epoch} | ")
         print(f"Training loss {epoch_loss} | ")
-        print(f"Validation loss {epoch_val_loss} | ")
+        print(f"Validation loss {epoch_val_loss / ((val_iter + 1) * batch_size)} | ")
```

Comment:
Can we get language for validation loss and training loss to match, fundamentally they should be calculated in similar ways. val_iter + 1 may equal num_training_sequences, but would be nice to just make sure this is accurate.

Reply:
100% agree, we need them to be computed in the exact same way